kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -22,15 +44,23 @@ export class JavascriptErrorReport extends ErrorReport {
     return {
       error_message: this.e.error.message,
       traceback: this.e.error.stack,
+      context: {
+        browser: this.getBrowserInfo(),
+        device: this.getDeviceInfo(),
+      },
```

Comment:
Same here https://github.com/learningequality/kolibri/pull/12382/files#r1662911798. Though we would just assign the context
```js
context: this.getContext()
```